Week1/homework-midy/exercise 2/index.js

```diff
@@ -0,0 +1,11 @@
+document.body.setAttribute('style', 'font-family: Arial, sans-serif;');
+
+document.getElementById('nickname').innerHTML = 'Midy';
+document.getElementById('fav-food').innerHTML = 'Pizza';
+document.getElementById('hometown').innerHTML = 'Heraklion';
+
+let lists = document.getElementsByTagName('li');
+
+for (let list in lists) {
+  lists[list].classList.add('list-item');
+}
```

Comment:
Again here it would be better to use for..of even if you had to convert lists into a js array eg using`Array.from(nodelist)`